app/routes/task_routes.py

```diff
@@ -1 +1,142 @@
-from flask import Blueprint
\ No newline at end of file
+from flask import Blueprint, abort, make_response, request, Response
+from app.models.task import Task
+from ..db import db
+from datetime import datetime
+import requests
+import os
+from dotenv import load_dotenv
+load_dotenv()
+
+tasks_bp = Blueprint("tasks_bp", __name__, url_prefix="/tasks")
+
+# slack URL
+SLACK_API_URL = os.environ.get('SLACK_API_URL')
+SLACK_API_TOKEN = os.environ.get('SLACK_API_TOCKEN')
+
+@tasks_bp.post("")
+def create_task():
+    request_body = request.get_json()
+    
+    try:
+        new_task = Task.from_dict(request_body)
+    
+    except KeyError as e:
+        response = {"details": "Invalid data"}
+        abort(make_response(response, 400))
+
+    db.session.add(new_task)
+    db.session.commit()
```

Comment:
Another place that is a sign for us to refactor this logic like in our previous create route.